networking/request-headers.html.md

```diff
@@ -20,9 +20,9 @@ Request headers carry information that is specific to the incoming request and i
 Not to be confused with the [environment variable](/docs/machines/runtime-environment/#fly_region) `FLY_REGION`, which is where the application is running.
 
 ### `X-Forwarded-For`
-**Client and Proxy List**: This is a comma separated list comprising of the client that originated the request and the proxy servers the request passed through. For example, "77.97.0.98, 77.83.142.33" contains the client and the one proxy it passed through.
+**Client and Proxy List**: A comma separated list of IP addresses including the address of the client that originated the request and the addresses of the proxy servers the request passed through. For example, "77.97.0.98, 77.83.142.33" contains the client and the one proxy it passed through.
 
-MDN has [full documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For) for this header.
+Follow the directives in the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For+external) for `X-Forwarded-For` to learn how to avoid selecting a spoofed IP address or a proxy service IP address as the client IP.
```

Comment:
I think we could make it clearer that blindly trusting this header is dangerous, and direct the user to use `Fly-Client-IP` if they don't want to deal with all this complexity, maybe:

> This header __must be treated with caution__ to avoid spoofing attempts. Follow the directives in the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For+external) to learn how to parse the header safely. If you aren't using another reverse proxy in front of the Fly.io platform, the `Fly-Client-IP` may be an easier choice to get access to client IP addresses.